code/lib/preview/package.json

```diff
@@ -61,6 +61,7 @@
     "@storybook/core-events": "workspace:*",
     "@storybook/global": "^5.0.0",
     "@storybook/preview-api": "workspace:*",
+    "browser-dtector": "^3.4.0",
```

Comment:
Happy to not use a lib and move [its logic](https://github.com/sibiraj-s/browser-dtector/blob/master/lib/browser-dtector.ts) to a single file if we want to save on bundle size

```
total
148K    code/node_modules/browser-dtector

breakdown
8.0K    code/node_modules/browser-dtector/CHANGELOG.md
4.0K    code/node_modules/browser-dtector/LICENSE
4.0K    code/node_modules/browser-dtector/README.md
8.0K    code/node_modules/browser-dtector/browser-dtector.esm.js
 16K    code/node_modules/browser-dtector/browser-dtector.esm.js.map
8.0K    code/node_modules/browser-dtector/browser-dtector.esm.min.js
8.0K    code/node_modules/browser-dtector/browser-dtector.esm.min.js.map
8.0K    code/node_modules/browser-dtector/browser-dtector.js
 16K    code/node_modules/browser-dtector/browser-dtector.js.map
8.0K    code/node_modules/browser-dtector/browser-dtector.min.js
8.0K    code/node_modules/browser-dtector/browser-dtector.min.js.map
 12K    code/node_modules/browser-dtector/browser-dtector.umd.js
 16K    code/node_modules/browser-dtector/browser-dtector.umd.js.map
8.0K    code/node_modules/browser-dtector/browser-dtector.umd.min.js
8.0K    code/node_modules/browser-dtector/browser-dtector.umd.min.js.map
4.0K    code/node_modules/browser-dtector/package.json
4.0K    code/node_modules/browser-dtector/types.d.ts
```

Reply:
It's 8kb, seems ok to me.

This is the source:
https://socket.dev/npm/package/browser-dtector/files/3.4.0/browser-dtector.esm.js

minified:
https://socket.dev/npm/package/browser-dtector/files/3.4.0/browser-dtector.esm.min.js

It seems the code is ES5 maybe, which might be compressed further if it was modernized a bit if we were going to copy it over, but I don't think that's necessary.